docs/miner/email_notification_guide.md

```diff
@@ -4,6 +4,13 @@ This guide describes how to configure and use the email notification feature in
 
 ---
 
+## How It Works
+
+If enabled, storage providers will receive email notifications for the following mining events:
+
+1. When a shard finishes syncing data via P2P and begins its mining task. This happens when a node is started for the first time or has been offline for a while, requiring its local storage to catch up with the network.  
```

Comment:
After the initial sync is completed, the node will rely on the downloader for data synchronization instead of the sync module